src/main/scala/Main.scala

```diff
@@ -75,6 +75,14 @@ object Main extends App {
         str.toInt
     }
 
+    /* default chunk size to 50 megs and make it user-changeable */
+    lazy val chunkSize = System.getProperty("chunkSize") match {
+      case null =>
+        50 * 1024 * 1024
+      case str: String =>
+        str.toInt * 1024 *1024
```

Comment:
Why not multiply by just 1048576?

Reply:
again, just to make it more explicit that we are going up two orders of magnitude. I'd normally write ^2 but couldn't remember the right syntax. I would expect the compiler to optimize this for runtime.